node/core/approval-voting/src/approval_checking.rs

```diff
@@ -843,11 +843,10 @@ mod tests {
 				no_show_duration,
 				needed_approvals,
 			),
-			RequiredTranches::Pending {
-				considered: 2,
-				next_no_show: None,
-				maximum_broadcast: 3,
-				clock_drift: block_tick,
+			RequiredTranches::Exact {
```

Comment:
Given the name of the test I suspect that this test is wrong if the result is not `Pending`